packages/core/src/login/webview/vue/amazonq/backend_amazonq.ts

```diff
@@ -31,7 +31,7 @@ export class AmazonQLoginWebview extends CommonAuthWebview {
         const importedApi = toolkitExt?.exports
```

Comment:
I wonder if we can do something like this:

```suggestion
        const importedApi = toolkitExt?.exports.getApi(VSCODE_EXTENSION_ID.amazonq)
```

That would avoid the need for an `extensionId` parameter on every function.